src/tripCard/specifications/index.md

```diff
@@ -0,0 +1,34 @@
+# Primary Button
+
+```js
+import { TripCard } from '@blablacar/ui-library'
```

Comment:
Add also a sample of usage (unless it's already the case with the story)

Reply:
It is